docs/getting-started/workflow-definition.mdx

```diff
@@ -176,6 +176,18 @@ Save the following token since it will not printed again:
 
 We have everything we need to integrate our CI with Chainloop!
 
-## Third-Party integrations
+## Optional setup
+
+### CAS Backend
+
+As part of an attestation process, you might want to collect different pieces of evidence such as Software Bill Of Materials (SBOMs), test results, runner logs, etc and then attach them to the final in-toto attestation. 
+
+By default, Chainloop comes pre-configured with what we call an `inline` backend. The inline backend **embeds** the pieces of evidence in the resulting attestations. This is useful to get started quickly but since the pieces of evidences are embedded in the attestation, their size is limited.
```

Comment:
pieces of evidences -> pieces of evidence
We have three "pieces of evidence" in a row. Should we use SSC metadata, just metadata or materials?

Reply:
I like it, thanks